util/test/demos/main.cpp

```diff
@@ -368,32 +369,35 @@ Usage: %s Test_Name [test_options]
       if(nk_begin(ctx, "Demo", nk_rect(0, 0, (float)width, (float)height), NK_WINDOW_NO_SCROLLBAR))
       {
         nk_layout_row_dynamic(ctx, 100, 1);
-        if(nk_group_begin(ctx, "Test Filter", NK_WINDOW_BORDER | NK_WINDOW_TITLE))
+        if(nk_group_begin(ctx, "Test Filter",
+                          NK_WINDOW_BORDER | NK_WINDOW_TITLE | NK_WINDOW_NO_SCROLLBAR))
         {
           nk_layout_row_dynamic(ctx, 30, ARRAY_COUNT(allow_names) + 1);
 
-          nk_label(ctx, "Filter tests:", NK_TEXT_LEFT);
+          nk_label(ctx, "API Filter:", NK_TEXT_LEFT);
 
+          nofilters = true;
           for(size_t i = 0; i < ARRAY_COUNT(allow); i++)
           {
             std::string text = allow_names[i];
 
             allow[i] = nk_check_label(ctx, text.c_str(), allow[i]) != 0;
+            nofilters &= allow[i];
```

Comment:
The intent of this change is fine but the way it's implemented leads to some confusing UI. The checkboxes are acting like exclusion checks but if all of the boxes are checked then nothing is excluded.

I think the behaviour is fine (default to everything, then check boxes to include them and filter out everything else) but you need to invert the sense of the checkboxes. They should start all unchecked instead of starting checked and checking them does exactly as it does today.

Reply:
Fixed. The Nuklear UI is confusing personally - the appearance of checked/unchecked boxes feels backwards to me. :P